package.json

```diff
@@ -31,17 +31,17 @@
     "build:esbuild": "node build.js",
     "build:typings": "tsc --project . --declaration",
     "build:watch": "gulp clean-out && pnpm copy:files && pnpm compile:less && concurrently \"tsc --project . --watch\" \"node build.js --watch\" \"gulp watch-less\"",
-    "check:all": "npm run check:prettier && npm run check:tsc && npm run check:lint",
-    "check:lint": "eslint .",
+    "check": "pnpm check:eslint && pnpm check:prettier && pnpm check:tsc",
```

Comment:
`check` is not a built-in `pnpm` command (unlike in yarn) so we can remove `:all`